jwst/extract_2d/nirspec.py

```diff
@@ -146,20 +146,19 @@ def set_slit_attributes(output_model, slit, xlo, xhi, ylo, yhi):
 
     Parameters
     ----------
-    output_model : `~jwst.datamodels.multislit.MultiSlitModel`
+    output_model : `~jwst.datamodels.SlitModel`
         The output model representing a slit.
     slit : namedtuple
         A `~jwst.transforms.models.Slit` object representing a slit.
     xlo, xhi, ylo, yhi : float
         Indices into the data array where extraction should be done.
         These are converted to "pixel indices" - the center of a pixel.
     """
-    xlo_ind, xhi_ind, ylo_ind, yhi_ind = _toindex((xlo, xhi, ylo, yhi)).astype(np.int16)
```

Comment:
This call to `_toindex` is superfluous as the inputs to this function have already been run through `_toindex` in the `offset_wcs()` function.